### ISSUE TITLE
Conference vs. journal

### ISSUE THREAD

**Issue by parno** [ieee-security/ongoing-submission-plan#8]
In Issue #7, @danwallach says:

> For lack of a better term, what S&P is building is a hybrid or "journfrence". While most university promotion & tenure committees have had this distinction in CS beaten out of them, it's technically feasible to label the resulting publications as "journal" publications rather than "conference". This has essentially no downside (maybe you have to reformat your paper in the single-page "journal" style) and lots of upside (looks good on the vita). Each point at which you release the newly accepted papers corresponds to an "issue" of the journal, and the set of all the papers to appear in any given year is a "volume". I'd encourage S&P to go with the "journal" aspect.

**Comment by parno** [comment]
I agree that we could quite reasonably call the output of this process a journal.  However, at this year's business meeting, some people raised concerns about changing to a journal after all the work that has been done to spread the word about the value of conferences for CS.  At present, unless there's a strong consensus on changing to a journal, my inclination is to decouple the change to the submission process from the decision as to whether we call it a conference or a journal.  In other words, we can change the submission process and continue to call it a conference, and then we can discuss changing the designation.

**Comment by danwallach** [comment]
The "journal" vs. "conference" distinction plays differently at different universities. I'm told that in Europe, it's a very big deal when coming up for promotion, and it's especially a big deal when you're in high-level meetings arguing about how much funding should go to CS versus going to other fields in science and engineering. They'll trot out all their fancy journal publications and demean our conference publications.

You may prefer to allow some sort of dual-naming scheme. Every _SIGGRAPH_ conference paper, for example, is also a paper in _ACM Transactions on Graphics_.
